plugins/dicom_viewer/webpack.helper.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(x) {
+    x.module.loaders.push({
+        test: /\.json$/,
+        loader: 'json',
+        include: ['node_modules'],
```

Comment:
We should use the local node_modules options that @ronichoudhury exposed so that we don't have to apply rules that may clobber girder core or other plugins.